### PR TITLE
removes the infrastructure client and kills the client application

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: elife/newrelic-infrastructure-removal.sls [hunk 1]
```diff
@@ -0,0 +1,10 @@
+newrelic-infrastructure-configuration:
+    file.absent:
+        - name: /etc/newrelic-infra.yml
+
+newrelic-infrastructure:
+    pkg.purged:
+        - name: newrelic-infra
+
+    service.dead:
```
Comment (giorgiosironi) [inline elife/newrelic-infrastructure-removal.sls]: maybe do first `service.dead` and then `pkg.purged`? Although `pkg.purged` may cover everything already

**Comment by lsh-0** [comment]
I wouldn't have thought it mattered but I think I've been on arch+systemd for too long. Installing a service and starting/stopping it are separate. apt ... unsure. I'll switch it's order.

**Comment by giorgiosironi** [comment]
The output of the build looks good, no permission denied errors.

**Comment by lsh-0** [comment]
it all went well, thanks for the builder fix

**Pull request merged**